refextract/references/kbs.py

```diff
@@ -42,6 +43,23 @@
 from ..documents.text import re_group_captured_multiple_space
 
 
+@contextlib.contextmanager
+def file_resolving(fpath, reader=None, **kwargs):
+    if isinstance(fpath, six.string_types):
+        print('Load kb path %s' % fpath)
```

Comment:
> I agree, although, in this PR @zzacharo was just refactoring.

True for this particular `print`, not true for all the other `print`s that he's adding below in the PR... I'd delete those. And this, honestly...